modern/base/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ubuntu:xenial
+ARG DISTRO_VERSION
+
+FROM ubuntu:${DISTRO_VERSION}
```

Comment:
xenial and 16.04 are tags, but same image. I'm fine use version instead of alias, because is easier to understand the distro age.